gerrit-entrypoint.sh

```diff
@@ -85,9 +85,15 @@ done
   #Customize gerrit.config
   set_gerrit_config repository."*".ownerGroup "Registered Users"
 
+  if [ -n "${DOWNLOAD_SCHEMES}" ]; then
+    set_gerrit_config --remove-section download || true
```

Comment:
Is there a way that only remove those download.scheme, instead of remove the whole download section?

Reply:
I'll try with `--unset-all download.scheme`.